generate-types.js

```diff
@@ -0,0 +1,60 @@
+const TypeWriter = require('@gimenete/type-writer')
+const webhooks = require('@octokit/webhooks-definitions/index.json')
```

Comment:
```suggestion
const webhooks = require('@octokit/webhooks-definitions')
```

That will allow to move the `index.json` file in the future. As long as the `main` property in `@octokit/webhooks-definitions`’s package.json file points to the new file it will continue to work